scripts/README.md

```diff
@@ -66,6 +79,64 @@ You can verify the installation success via command line or a browser.
    - Check the Core Admin menu for the CKAN core with `instanceDir /var/solr/data/ckan`.
    - Verify that the schema properties reflect the Ontario theme (Unique Key Field should be `index_id`).
 
+## CKAN & Ontario Theme
+
+Bash script: `setup_ckan_local.sh`
+
+This bash script installs CKAN and Ontario Theme locally, preferably on a Virutal Machine. The script follow the steps from the [official installation guide for installing CKAN from source](https://docs.ckan.org/en/2.9/maintaining/installing/install-from-source.htmlhttps:/).
+
+### Developer and Production Modes
+
+CKAN can be installed in Developer Mode, where the CKAN server is started manually through the commandline (`$ ckan -c /etc/ckan/default/ckan.ini run `) and is available at `localhost:5000` in the browser; Or CKAN can be installed in Production Mode where `supervisor` application is responsible for running the CKAN application and `nginx` is used as the webserver available directly at `localhost` in the browser. The Production Mode is how CKAN runs in production environment.
+
+The script installs CKAN+Ontario Theme in Production Mode.
+
+### What does the Script need?
+
+The script needs Solr, postgreSQL and relevant databases to be setup. These steps are recorded in the instructions below.
+
+For configuration, files in `config/` directory are used.
+
+* `config/ckan/ckan.ini` file is copied to`/etc/ckan/default/ckan.ini`, and modified during the course of execution of the script
+* `config/gtm/gtm.html` and `config/gtm/gtm_ns.html` are copied to `/usr/lib/ckan/default/src/ckanext-ontario_theme/ckanext/ontario_theme/templates/internal/`
+* `config/nginx/local_ckan_ssl` is copied to `/etc/nginx/sites-available/`
+* `config/supervisor/ckan-uwsgi.conf` and `config/supervisor/ckan-worker.conf` are copied to `/etc/supervisor/conf.d/`
+
+### Running the script
+
+**Command line:**
+
+1. Do the steps common to the scripts in [common steps](#common-steps)
+2. Setup Solr8 locally as described in [Solr Installation](#solr-installation)
+
+3. Setup PostgreSQL locally
+
+```bash
+sudo bash setup_postgres_local.sh
+```
```

Comment:
I would either add the export sudopass line before this or remove the unset sudopass from line 55-58 (the solr step)